src/partials/id_verification_device.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="card">
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo"/>
+  </div>
+  <form class="form form--spacing-lg" id="AuthnWidgetForm" style="margin-left:27px;margin-right:27px;">
+    <div class="stack stack--small">
+      <h1 id="header" class="heading" data-id="heading">Verification Required</h1>
+      <h4 id="description" class="heading heading--4 text-block" data-id="heading">
+        If you are already on your mobile device, select This Device to open a web link on your mobile device to start the verification process, otherwise select the Other Device button.
+      </h4>
+    </div>
+    <div class="org-logo" style="margin:30px">
+      <img id="icon" class="org-logo__image" src="" alt="New Tab"/>
+    </div>
+    <table style="width:100%;">
```

Comment:
Is this what the PF templates look like? Generally we moved away from tables and use the end-user.css to format pages